install_config/topics/configuring_for_vsphere_storage.adoc

```diff
@@ -0,0 +1,191 @@
+
+[[vsphere-creating-persistent-volume]]
+=== Creating persistent volumes
+
+. Define a PV object definition, for example *_vsphere-pv.yaml_*:
++
+[source, yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv0001 <1>
+spec:
+  capacity:
+    storage: 2Gi <2>
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  vsphereVolume: <3>
+    volumePath: "[datastore1] volumes/myDisk" <4>
+    fsType: ext4 <5>
+----
+<1> The name of the volume. This must be how it is identified by xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[PV claims] or from pods.
+<2> The amount of storage allocated to this volume.
+<3> The volume type being used. This example uses `vsphereVolume`, and the label is used to mount a vSphere VMDK volume into pods. The contents of a volume are preserved when it is unmounted. The volume type supports VMFS and VSAN datastore.
+<4> This VMDK volume must exist, and you must include brackets ([]) in the volume definition.
+<5> The file system type to mount. For example, `ext4`, `xfs`, or other file-systems.
+
+[IMPORTANT]
+====
+Changing the value of the `fsType` parameter after the volume is formatted and
+provisioned can result in data loss and pod failure.
+====
+
+. Create the PV:
++
+[source, bash]
+----
+$ oc create -f vsphere-pv.yaml
+  persistentvolume "pv0001" created
+----
+
+. Verify that the PV was created:
++
+[source, bash]
+----
+$ oc get pv
+NAME    LABELS  CAPACITY  ACCESSMODES   STATUS    CLAIM   REASON  AGE
+pv0001  <none>  2Gi       RWO           Available                 2s
+----
+
+Now you can
+xref:../../dev_guide/persistent_volumes.adoc#dev-guide-persistent-volumes[request
+storage using PV claims], which can now use your PV.
+
+[IMPORTANT]
+====
+PV claims only exist in the user's namespace and can only be referenced by a pod
+within that same namespace. Any attempt to access a PV from a different
+namespace causes the pod to fail.
+====
+
+[[volume-format-vsphere]]
+=== Formatting VMware vSphere volumes
+
+Before {product-title} mounts the volume and passes it to a container, it checks
+that the volume contains a file system as specified by the `fsType` parameter in
+the PV definition. If the device is not formatted with the file
+system, all data from the device is erased, and the device is automatically
+formatted with the given file system.
+
+This allows unformatted vSphere volumes to be used as PVs, because
+{product-title} formats them before the first use.
+
+
+== Provisioning VMware vSphere volumes via a Storage Class
+
+. {product-title} creates the following `storageclass` when you use the `vsphere-volume`
+provisioner and if you use the `openshift_cloudprovider_kind=vsphere` and
+`openshift_vsphere_*` variables in the Ansible inventory. Otherwise, you can create
+it manually:
++
+[source,yaml]
+----
+$ oc get --export storageclass standard -o yaml
```

Comment:
The name `standard` here should be replaced with `vsphere-standard` to be consistent.